BondageClub/Assets/Female3DCG/Female3DCGExtended.js

```diff
@@ -2933,6 +2933,27 @@ var AssetFemale3DCGExtended = {
 				},
 			},
 		}, // LatexCorset1
+		HeavyLatexCorset: {
+			Archetype: ExtendedArchetype.TYPED,
+			Config: {
+				Options: [
+					{
+						Name: "Normal",
+						Property: { Type: null },
+					},
+					{
+						Name: "Straps",
+						Property: { Type: "Straps" },
+					},
+				],
+				Dialog: {
+					Load: "SelectHeavyLatexCorsetType",
+					TypePrefix: "HeavyLatexCorsetType",
+					ChatPrefix: "HeavyLatexCorsetSet",
+					NpcPrefix: "HeavyLatexCorset",
+				},
+			}
+		}, // HeavyLatexCorset
```

Comment:
If this config is identical to the existing Heavy Latex Corset, you can use the `CopyConfig` property to duplicate configuration between groups:
```suggestion
		HeavyLatexCorset: {
			Archetype: ExtendedArchetype.TYPED,
			CopyConfig: { GroupName: "ItemTorso", AssetName: "HeavyLatexCorset" },
		}, // HeavyLatexCorset
```

Reply:
It isn't completely identical mainly I removed the chat tags from the corset version since it doesn't need them. As cloth type items don't send a chat message. I am not sure if it matters. If it doesn't then I can commit what you are suggesting.

Reply:
I don't believe that matters off the top of my head - I'd recommend trying it out. If we can get away with trimming the config down, I think we should.

Reply:
Ok I am going to commit it and test it out. I doubt it will matter. Since if it isn't sending the message in the first place I doubt the chat tags will matter. Which means I am now also going to end up doing this for the Suits. Since then I only need 2.

Reply:
Ok tested it now there is no difference I can see it can stay merged. This should be all done.